src/Perl6/Actions.nqp

```diff
@@ -1821,32 +1807,20 @@ class Perl6::Actions is HLL::Actions does STDActions {
                 ),
             );
         }
-        $past.push(QAST::Op.new(
-            :op('callmethod'), :name('merge-symbols'),
-            QAST::Op.new(
-                :op('who'),
-                $*W.symbol_lookup(['GLOBAL'], $/),
-            ),
-            QAST::Op.new(
-                :op('who'),
-                QAST::Op.new(
-                    :op('callmethod'), :name('globalish-package'),
-                    QAST::Op.new(
-                        :op('callmethod'), :name('handle'),
-                        $op,
-                    ),
-                ),
-            ),
-        ));
+        my $require_past := QAST::Op.new(:node($/), :op<call>,
```

Comment:
`$import_past` renamed to `$require_past` and calls `REQUIRE_IMPORT` regardless of whether there are args to `require` because GLOBALish merging has been moved there.